examples/serre_green_naghdi_1d/serre_green_naghdi_manufactured.jl

```diff
@@ -0,0 +1,54 @@
+using OrdinaryDiffEqTsit5
+using DispersiveShallowWater
+using SummationByPartsOperators: upwind_operators, periodic_derivative_operator
+
+###############################################################################
+# Semidiscretization of the Serre-Green-Naghdi equations
+
+# for now only bathymetry_flat is implemented (bathymetry_variable soon)
+equations = SerreGreenNaghdiEquations1D(bathymetry_type = bathymetry_flat,
+                                        gravity = 9.81)
+
+initial_condition = initial_condition_manufactured
+source_terms = source_terms_manufactured
+boundary_conditions = boundary_condition_periodic
+
+# create homogeneous mesh
+coordinates_min = 0.0
+coordinates_max = 1.0
+N = 256
+mesh = Mesh1D(coordinates_min, coordinates_max, N)
+
+#= for testing rhs_sgn_upwind! use:
```

Comment:
```suggestion
#= for testing upwind operators use:
```
Since the name `rhs_sgn_upwind!` is internal, I wouldn't expose it to the user.